profiles/dfs_tec/config/install/page_manager.page_variant.services.yml

```diff
@@ -27,7 +27,7 @@ variant_settings:
         max_age: '-1'
       region: secondary
       weight: '-9'
-      classes: services-view
+      css_classes: services-view
```

Comment:
I feel like this was the real use case where we needed it and throwing generic classes on blocks in the config/install files was misguided.